### DISCUSSION TITLE
参考文献列表是否可以分成原始文献和研究文献两部分

### DISCUSSION THREAD

**Discussion by SYQsb** [tuna/thuthesis#953] [Q&A] [answered]
人文学院的论文一般会参考一些一手史料，通常在条目较多的情况下，文献列表中会将一手文献和二手文献分开陈列，也就是分成两个小标题，内部还是首字母排序等形式。请问目前的模板是否可以实现这一功能？

**Comment by Harry-Chen** [comment, marked as answer]
biblatex 原生支持这样的功能，可以给每篇参考文献增加 `keyword` 字段，并使用如下的命令：

```latex
\printbibliography[keyword={foo},title={Foo-related only}]
\printbibliography[keyword={bar},title={Bar-related only}]
```

**Reply by SYQsb** [reply]
感谢回复！请问bibtex是否也存在这样的功能呢？我使用的命令是\usepackage{natbib}
\bibliographystyle{thuthesis-author-year}

**Reply by zepinglee** [reply]
> 感谢回复！请问bibtex是否也存在这样的功能呢？我使用的命令是\usepackage{natbib} \bibliographystyle{thuthesis-author-year}

bibtex 方案无法实现这个功能。

**Reply by Harry-Chen** [reply]
thuthesis 同时支持 bibtex 和 biblatex，所以更改一下使用的方案就行了
